include/open62541/architecture_definitions.h

```diff
@@ -499,7 +505,7 @@ static UA_INLINE size_t
 UA_atomic_subSize(volatile size_t *addr, size_t decrease) {
 #if UA_MULTITHREADING >= 100
 #ifdef _MSC_VER /* Visual Studio */
-    return _InterlockedExchangeSub(addr, decrease) - decrease;
+    return InterlockedExchangeAdd((LONG*) addr, - (LONG) decrease) - decrease;
```

Comment:
The second `(LONG)` is required.

The first one is not required for 64-bit build, I would have to check the 32-bit.